Logger/BaseLoggerMixins.cs

```diff
@@ -1,7 +1,71 @@
-﻿namespace Logger
+﻿using System;
+
+namespace Logger
 {
     public static class BaseLoggerMixins
     {
-        
+        public static void Error(this BaseLogger baseLogger, string message, params object[] args)
+        {
+            if (baseLogger is null)
+            {
+                throw new ArgumentNullException("baseLogger is null");
```

Comment:
The string parameter to `ArgumentNullException` is expected to the [name of the argument](https://docs.microsoft.com/en-us/dotnet/api/system.argumentnullexception.-ctor?view=netcore-3.0#System_ArgumentNullException__ctor_System_String_) that was null.
```suggestion
                throw new ArgumentNullException(nameof(baseLogger));
```